funnel/templates/proposals.html.jinja2

```diff
@@ -76,15 +76,7 @@
   <div class="mui-container">
     <div class="page-content page-content--mob-nav">
       <div class="grid">
-        <div class="grid__col-xs-12">
-          {% if current_auth.is_authenticated and project.current_roles.admin %}
-            {{ admin_panel(project, cfp_transition_form=cfp_transition_form) }}
-          {%- endif %}
-        </div>
-      </div>
-
-      <div class="grid">
-       {% if project.current_roles.admin or project.current_roles.reviewer or not project.cfp_state.NONE -%}
+       {% if project.current_roles.profile_admin or project.current_roles.reviewer or not project.cfp_state.NONE -%}
```

Comment:
1. Remove `profile_admin` here.
2. Projects have an editor role, but not a reviewer role.